src/cms/forms/events/event_filter_form.py

```diff
@@ -62,3 +62,12 @@ class EventFilterForm(forms.Form):
     )
 
     poi_id = forms.IntegerField(widget=forms.HiddenInput, initial=-1, required=False)
+
+    query = forms.CharField(required=False)
+
+    def filters_visible(self):
```

Comment:
I'd add a `@property` decorator here.

Reply:
Just out of curiosity, I checked in which cases Django uses the `@property` decorator for forms, but I couldn't find a consistent pattern :sweat_smile: 
For example, similar form-methods like `form.is_valid()`, `form.has_changed()` or `non_field_errors()` are methods, but `form.errors` and `form.changed_data` are properties...